cap_anndata/cap_anndata.py

```diff
@@ -277,6 +307,9 @@ def _link_obsm(self) -> None:
                     # sparse array
                     self._obsm[entity_name] = ad.experimental.sparse_dataset(entity)
 
+    def layer_keys(self) -> List[str]:
+        return list(self.layers.keys())
```

Comment:
we don't actually need this method as the original AnnData doesn't have it. It's not a problem we can keep it, but please rename it to `layers_keys`

Reply:
removed